wolox-books/src/components/BooksList.vue

```diff
@@ -18,12 +18,15 @@ export default {
     }
   },
   methods: {
+    ...mapActions(['bringBooks']),
     getBooks() {
-      getListOfBooks().then(response => {
-        this.booksToShow = response.data
-      })
+      this.bringBooks()
+      this.booksToShow = this.allBooks
```

Comment:
No need to do this since you are already mapping the store with your state :smile: